.vimrc

```diff
@@ -50,14 +50,61 @@
         endif
     " }
 
+    " Config files {
+        " All config variables handle invisibility for you.
+        " This cuts down on a lot of `if` statements in all our startup files.
+        " So, use them like: g:spf13_vim_cache_home . 'foo/bar'
+        "               NOT: g:spf13_vim_cache_home . '.foo/bar'
+
+        let g:spf13_vimrc = $HOME . '.vimrc'
+        let g:spf13_gvimrc_local = $HOME . '.gvimrc.local'
+        " Used for: vimrc.* files
+        if $XDG_CONFIG_HOME != ""
```

Comment:
I check for the variables this way (instead of other ways which usually just check to see if an environment variable exists by that name, in case someone has set `$XDG_CONFIG_HOME=""` etc. for some reason (probably temporarily) to keep us from attempting to write to /vim or something without root permissions.